okhttp/src/main/java/okhttp3/OkHttpClient.java

```diff
@@ -844,15 +844,25 @@ public Builder dispatcher(Dispatcher dispatcher) {
      * HTTP/1.1} only. If the server responds with {@code HTTP/1.0}, that will be exposed by {@link
      * Response#protocol()}.
      *
-     * @param protocols the protocols to use, in order of preference. The list must contain {@link
-     * Protocol#HTTP_1_1}. It must not contain null or {@link Protocol#HTTP_1_0}.
+     * @param protocols the protocols to use, in order of preference. If the list contains {@link
```

Comment:
Could we reword this to start with the normal case of HTTP_1_1.  As it is has a very H2C centric view of the world.

Reply:
I'll take another crack. I started trying to word it that way, but ran into issues with clarity.